### DISCUSSION TITLE
GWT plugin 4.1.0 RC2 is out for testing

### DISCUSSION THREAD

**Discussion by protoism** [gwt-plugins/gwt-eclipse-plugin#494] [Announcements]
We're almost done!
Hopefully this is going to be the release version (hence the URL)
So... if no problems show up, in a couple of days we'll publish to the marketplace
As usual, testing is very, very, very appreciated. 
Please report any problem in this thread (or link an issue)

Update site is here:

https://plugins.gwtproject.org/eclipse/gwt-eclipse-plugin/4.1.0/

It's a link to an update site, instructions here

https://wiki.eclipse.org/Eclipse_Project_Update_Sites

Thanks to @keinhaar @niloc132 for support in preparing this release

**Comment by keinhaar** [comment]
I checked
- eclipse 2023-09, 2023-12, 2024-03 and 2024-06
- installing the plugin only (no SDK's)
- installing with one or more SDK's
- installed the newly added 2.10.1 and 2.11 SDK's
- created and ran the sample application in all eclipse versions
- created and ran the sample application with any of the provided SDK's and with my local 2.10 installation.

Results:
- Installation is just fine now in all versions
- Sample App runs fine
- All Dialogs and Menus look normal to me.

Only in eclipse 2024-06 i had to change the JDK to JAVA 17 or below, to successfully run the sample app, which is no fault of the plugin.

Conclusion:
Seems to be ready for release, but maybe we give some other testers a (short) chance to test :-)

**Reply by protoism** [reply]
Thanks @keinhaar, much appreciated.
I was about to publish on the marketplace, let's delay until this evening.
I'd mark compatibility from 2023-09 onwards, what do you think?

**Comment by keinhaar** [comment]
I didn't test older versions, but i think it will be compatible up to the version where the maven incompatibility occured. will it be installable in older versions if you mark 2023-09+ ?
anyway: 4 versions back, should be enough.

**Comment by protoism** [comment]
Published on marketplace.
Let's see how long it takes.
